### ISSUE TITLE
Introductions + MLPs, Gradient Descent & Backpropagation

### ISSUE THREAD

**Issue by Divya9Sasidharan** [Divya9Sasidharan/Data-Science#1]
Start the MLP and Tensorflow /Keras study
